lib/attributes/char.rb

```diff
@@ -9,6 +9,10 @@ def Char.name
     XMLData.name
   end
 
+  def Char.stance
+    XMLData.stance_text
+  end
```

Comment:
This should likely be:
```
  def Char.stance(*args)
    checkstance(*args)
  end
```

Reply:
I couldn't find the original overload for `Char.stance` in 5.6.2 to compare to, but this would make a bit more sense to support the duality of using either `Char.stance` or `checkstance`